clang/lib/Driver/Driver.cpp

```diff
@@ -1022,7 +1022,7 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
 
       for (const auto &TripleAndArchs : DerivedArchs)
         OpenMPTriples.insert(TripleAndArchs.first());
-    }
+    } // end of offload-arch
```

Comment:
I'm assuming this addition was for your reference and not part of your new implementation.